### ISSUE TITLE
synapse 1.49.0 update deleted my productive server config

### ISSUE THREAD

**Issue by freshprince** [void-linux/xbps#441]
```
post_install() {
	vsv synapse

	vinstall docs/sample_config.yaml 644 etc/synapse homeserver.yaml
	vinstall docs/sample_log_config.yaml 644 etc/synapse log.yaml
}
```

not really a good idea if someone already has a productive config in /etc/synapse/homeserver.yaml :(

**Comment by Chocimier** [comment]
Duplicate of #342, fix at #426.
Should be fixed in void packages anyway, see https://github.com/void-linux/void-packages/pull/34665

**Comment by freshprince** [comment]
thanks. for completeness: void-linux/void-packages#34531 is the PR that introduced this behaviour.
